### DISCUSSION TITLE
Change osu!catch's HP system to be cumulative (similar to taiko)

### DISCUSSION THREAD

**Discussion by sverrier** [ppy/osu#13127] [Ideas]
Movement in osu!catch is hard capped at a certain speed, and on anything higher starred you rely on hyper fruits to even follow the beatmap. Because of this, as soon as you miss in these maps it becomes extremely hard, often entirely impossible to recover. This leads to a very brutal pass/fail system that can pretty frustrating. 

If the system worked similar to taiko, where you have a set target to reach with your performance throughout the map, while mistakes would still be hard to recover from you would have the chance to make up for it, as well as not being instantly kicked from the map.

**Comment by bdach** [comment]
As far as I understand the desired direction was actually the opposite (towards unification of all HP implementations across rulesets to the osu! standard, which would mean deprecating the taiko HP system).

This requires extensive discussion with the community regardless, which this is probably not the right place for.

**Comment by sverrier** [comment]
Understandable. I would say that if any mode should have it's own HP system it is catch though, simply because it's the only mode where your input is hard capped. There simply is no recovering in many cases.
